WordPress/Classes/ViewRelated/Site Creation/WebAddressStep/AddressCell.swift

```diff
@@ -0,0 +1,24 @@
+import UIKit
+
+final class AddressCell: UITableViewCell, ModelSettableCell {
```

Comment:
I was going to ask if we could reuse the existing cell in Domains, but [it doesn't appear](https://github.com/wordpress-mobile/WordPress-iOS/blob/2d4089e965c7db2c3f458873278d3e1b17bf558d/WordPress/Classes/ViewRelated/NUX/DomainSuggestionsTableViewController.swift#L270-L278) that there is one.